locales/es.json

```diff
@@ -1,7 +1,8 @@
 {
   "mainHeader": {
     "missions": "Misiones",
-    "more": "Más"
```

Comment:
Can you remove this file from this PR? It has nothing to do here, I guess it's because the base branch is wrong (the PR should be against `dev`).

Reply:
I can rebase the branch with dev.

Reply:
Looks like it is up-to-date with `dev`.
Maybe the diff was because the base branch to merge was `master`